README.md

```diff
@@ -1,23 +1,21 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My Portfolio
 
-> One paragraph statement about the project.
+> This is the first project of my portfolie. It has no functionally as yet.
 
-![screenshot](./app_screenshot.png)
+![screenshot](../image/Illustration.png)
 
 Additional description about the project and its features.
```

Comment:
- [ ]Please consider removing this text To make your readme look professional as requested by the previous code reviewer.